packages/bpk-component-datepicker/src/BpkDatepicker.js

```diff
@@ -313,6 +325,7 @@ BpkDatepicker.propTypes = {
 
 BpkDatepicker.defaultProps = {
   calendarComponent: BpkCalendar,
+  InputComponent: null,
```

Comment:
should this not start with a lowercase 'i' as it is a prop?

Reply:
Good spot this would have been from autocomplete in my editor